### ISSUE TITLE
[SPProductUpdate, SPInstall, SPInstallLanguagePack] Installation hangs when installing from an UNC path

### ISSUE THREAD

**Issue by ykuijs** [PowerShell/SharePointDsc#1035]
#### Details of the scenario you tried and the problem that is occurring
When specifying an UNC path to install SharePoint, a Language Pack or CU, the installation hangs endlessly.

#### Verbose logs showing the problem
N/A

#### Suggested solution to the issue
I have investigated this issue and discovered that Start-Process gives a security warning when a UNC path is used. This can be resolved by adding the used servername to the IE Local Intranet Zone:
https://www.reddit.com/r/PowerShell/comments/7d49z3/start_script_from_uncpath_without_security_warning/
https://technochat.in/2008/12/disable-unc-path-security-warning-from-group-policy/

This can be resolved via PowerShell by executing:
```powershell
$null = New-Item -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\Domains\$serverName"
Set-ItemProperty -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\Domains\$serverName" -Name file -Value 1 -Type DWord
$null = New-Item -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\EscDomains\$serverName"
Set-ItemProperty -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\EscDomains\$serverName" -Name file -Value 1 -Type DWord
```
#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
SPInstall "Install_SharePoint"
{
    IsSingleInstance     = "Yes"
    BinaryDir            = "\\server\install\sp\setup.exe"
    ProductKey           = XXXXX-XXXXX-XXXXX-XXXXX-XXXXX
    InstallPath          = "C:\SP_Install"
    DataPath             = "C:\SP_Data"
    PSDscRunAsCredential = $InstallAccount
}
```

#### The operating system the target node is running
Windows Server 2016

#### Version of SharePoint that is used (e.g. SharePoint 2016)
SharePoint 2019

#### Version and build of PowerShell the target node is running
v5.1

#### Version of the DSC module that was used ('dev' if using current dev branch)
dev

**Comment by ykuijs** [comment]
Also update Readme files. Some of them specify that UAC has to be disabled, which is not the case.

Fix will be included in my next bugfix PR

**Comment by ykuijs** [comment]
Fixed in PR #1036

**Comment by ykuijs** [comment]
This check also needs to get added to SPInstallPrereqs. Will be included in my next Bugfix PR
